Style.css

```diff
@@ -0,0 +1,3 @@
+h2 {
+    color:greenyellow;
+}
```

Comment:
Please remove the extra line at the end of this file for code consistency and to keep up with best practice.